modules/bmo-verifying-the-hostfirmware-settings-resource-is-valid.adoc

```diff
@@ -0,0 +1,27 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="verifying-the-hostfirmware-settings-resource-is-valid_{context}"]
+= Verifying the HostFirmware Settings resource is valid
+
+After editing the `spec.settings` section of the `HostFirmwareSettings` resource, the Bare Metal Operator will validate the resource. You can use the following procedure to verify that the resource is valid.
+
+.Procedure
+
+. Get a list of hosts:
++
+[source,terminal]
+----
+$ oc get hfs -n openshift-machine-api
+----
+
+. Verify that the `HostFirmwareSettings` resource for a particular host is valid:
++
+[source,terminal]
+----
+$ oc describe hfs/<host_name> -n openshift-machine-api
+----
++
+Where `<host_name>` is the name of the host. If the response returns `ValidationFailed`, there is an error in the resource configuration. If there is an error, update the values to conform to the schema as described in the "About the FirmwareSchema resource" section.
```

Comment:
Do we really need the second line here?

```suggestion
Where `<host_name>` is the name of the host.
```

Reply:
What you could do, is add an xref directly after the module `include::` reference in the assembly to link directly to the topic you want to refer to.